extensions/http-curl/tests/HTTPIntegrationBase.h

```diff
@@ -91,4 +87,64 @@ void CoapIntegrationBase::setUrl(std::string url, CivetHandler *handler) {
   }
 }
 
+class VerifyC2Base : public CoapIntegrationBase {
+ public:
+  explicit VerifyC2Base(bool isSecure)
+      : isSecure(isSecure) {
+  }
+
+  virtual void testSetup() {
+    LogTestController::getInstance().setDebug<utils::HTTPClient>();
+    LogTestController::getInstance().setDebug<LogTestController>();
+  }
+
+  void runAssertions() {
```

Comment:
This should be defined by any derived class (to make sure they test something), so I think you can delete it.
That has the advantage of making this baseclass abstract.